src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java

```diff
@@ -463,13 +481,29 @@ private void updateCitizens()
     }
 
     /**
+     * Re-sorts the WorkOrders list according to the priorities inside the list.
+     */
+    private void sortWorkOrders()
+    {
+        workOrders.sort((first, second) -> {
+            if (second.getPriority() > first.getPriority() == true) {
+                return 1;
+            } else if (second.getPriority() < first.getPriority() == true) {
+                return -1;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 13. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AIndentationCheck)